how-to-win-friends.md

```diff
@@ -0,0 +1,33 @@
+# Book Report
+
+## Title
+
+how to win friends and influence people.
```

Comment:
The title should be written like "How to Win Friends & Influence People". Got that from amazon.com.